### PR TITLE
airgap upload

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: ffi/airgap.go [hunk 1]
```diff
@@ -21,21 +23,90 @@ import (
...
+
+//export PullFromAirgap
+func PullFromAirgap(licenseData, airgapDir, downstream, outputFile, registryHost, registryNamesapce string) int {
+	workspace, err := ioutil.TempDir("", "kots-airgap")
 	if err != nil {
-		fmt.Printf("failed to download airgap archive: %s\n", err)
+		fmt.Printf("failed to create temp dir: %s\n", err)
 		return 1
 	}
+	// defer os.RemoveAll(workspace)
```
Comment (marccampbell) [inline ffi/airgap.go]: should this be put back in?

File: pkg/image/image.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	"path"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"sigs.k8s.io/kustomize/v3/pkg/image"
+)
+
+func BuildRewriteList(rootDir string, host string, namespace string) ([]image.Image, error) {
+	images, err := findImages(rootDir, host, namespace, []string{})
+	return images, errors.Wrapf(err, "failed to find images")
```
Comment (marccampbell) [inline pkg/image/image.go]: why does this return an error always?

File: ffi/airgap.go (hunk 1 above)
Comment (divolgin) [reply ffi/airgap.go]: yeah

File: pkg/image/image.go (hunk 2 above)
Comment (divolgin) [reply pkg/image/image.go]: errors.Wrap will return nil when there is no error... it does feel a bit odd without the if

**Pull request merged**